core/pkg/server/sender.go

```diff
@@ -1266,7 +1266,13 @@ func (s *Sender) sendRequestLogArtifact(record *service.Record, msg *service.Log
 func (s *Sender) sendRequestDownloadArtifact(record *service.Record, msg *service.DownloadArtifactRequest) {
 	var response service.DownloadArtifactResponse
 
-	if err := artifacts.NewArtifactDownloader(
+	// Perform the nil check on the GraphQL client.
```

Comment:
again this comment is redundant it is obvious from the code

Reply:
Changed